internal/background/initialize.go

```diff
@@ -13,11 +13,22 @@ import (
 // incoming requests to overload the sender.
 const availabilityStatusBatchSize = 1024
 
-// Initialize starts background goroutines. Use context cancellation to stop them.
-func Initialize(ctx context.Context) {
+// InitializeApi starts background goroutines for REST API processes.
+// Use context cancellation to stop them.
+func InitializeApi(ctx context.Context) {
 	logger := ctxval.Logger(ctx).With().Bool("background", true).Logger()
 	ctx = ctxval.WithLogger(ctx, &logger)
 
 	// start availability request batch sender
 	go sendAvailabilityRequestMessages(ctx, availabilityStatusBatchSize, 5*time.Second)
 }
+
+// InitializeWorker starts background goroutines for worker processes.
+// Use context cancellation to stop them.
+func InitializeWorker(ctx context.Context, workerName string) {
+	logger := ctxval.Logger(ctx).With().Bool("background", true).Logger()
```

Comment:
```suggestion
	logger := ctxval.Logger(ctx).With().Bool("worker", true).Logger()
```

Reply:
I know this is confusing, but I would like to keep this as "background" to indicate this comes from the "background" goroutine. We have two now:

* Enqueuer for Kafka messages for checks, runs within "api" pods.
* This for jq metrics, runs within "worker" pods.

The package is named "background" so that was the idea.

Reply:
oki :shrug: :+1: